src/lib.rs

```diff
@@ -228,7 +228,7 @@ mod error;
 mod into_url;
 mod response;
 
-pub use self::error::{Error, Result};
+pub use self::error::{BoxError, Error, Result};
```

Comment:
I wouldn't bother to re-export this type alias. It's pretty common for people to type out the full type themselves, and (in my opinion) this is more noise than signal for the reqwest docs.

Reply:
Updated.